app/schema_validation/__init__.py

```diff
@@ -16,13 +18,11 @@ def build_error_message(errors, schema):
         field = "'{}' {}".format(e.path[0], e.schema.get('validationMessage')) if e.schema.get(
             'validationMessage') else e.message
         s = field.split("'")
-        field = {s[1]: s[2].strip()}
+        field = OrderedDict({"error": "ValidationError", "message": "{}{}".format(s[1], s[2])})
```

Comment:
if you really want to sort the keys (which i don't think is especially worthwhile as it'll be an unformatted blob, which when is read in will lose ordering), then you can call `sort_keys=True` on json.dumps


```
>>> json.dumps(x)
'{"b": 1, "a": 1, "c": 1}'
>>> json.dumps(x, sort_keys=True)
'{"a": 1, "b": 1, "c": 1}'
```